### PR TITLE
Hack __text_signature__ back to working with abi3

### PR DESCRIPTION
**Pull request by alex**

### CODE REVIEW

**Comment by alex** [comment]
This gross hack seems to break PyPy -- not a big surprise. I guess I'll just disable it there.

File: tests/test_text_signature.rs [hunk 1]
```diff
@@ -72,7 +70,7 @@ fn class_with_docs_and_signature() {
 // Ignored because heap types don't have working __text_signature__:
 // https://github.com/python/cpython/blob/master/Objects/typeobject.c#L864-L870
```
Comment (kngwyu) [inline tests/test_text_signature.rs:71]: It's also outdated?

**Review by kngwyu — APPROVED** [review]
Thanks, looks nice as a remedy :+1:

File: tests/test_text_signature.rs (hunk 1 above)
Comment (alex) [reply tests/test_text_signature.rs]: Ooops. Fixed.

**Pull request merged**